### PR TITLE
Warning added for baking a migration with no valid action prefix

### PR DESCRIPTION
**Pull request by rrd108**
@dereuromark Did you had something like this for #652?

### CODE REVIEW

**Comment by dereuromark** [comment]
Yes, that sounds probably useful to avoid no-ops.

File: src/Command/BakeMigrationCommand.php [hunk 1]
```diff
@@ -76,8 +76,21 @@ public function templateData(Arguments $arguments): array
...
+        $arguments = $arguments->getArguments();
+        unset($arguments[0]);
+        $columnParser = new ColumnParser();
+        $fields = $columnParser->parseFields($arguments);
+        $indexes = $columnParser->parseIndexes($arguments);
+        $primaryKey = $columnParser->parsePrimaryKey($arguments);
+
         $action = $this->detectAction($className);
 
+        if (empty($action) && count($fields)) {
```
Comment (dereuromark) [inline src/Command/BakeMigrationCommand.php]: if (!$action) is less cloaky.

File: src/Command/BakeMigrationCommand.php [hunk 2]
```diff
@@ -76,8 +76,21 @@ public function templateData(Arguments $arguments): array
...
+            /** @psalm-suppress PossiblyNullReference */
+            $this->io->abort('Migration name should start with one of the following prefixes: Create, Drop, Add, Remove, Alter. See: https://book.cakephp.org/migrations/4/en/index.html#migrations-file-name');
+        }
+
         if (empty($action)) {
```
Comment (dereuromark) [inline src/Command/BakeMigrationCommand.php:93]: Same here I guess

File: tests/TestCase/Command/BakeMigrationCommandTest.php [hunk 3]
```diff
@@ -328,4 +328,17 @@ public function testDetectAction()
...
+     * Test migration without Create, Drop, Add, Remove, Alter prefix
+     *
+     * @return void
+     */
+    public function testActionWithoutValidPrefix()
+    {
+        $this->exec('bake migration SleepUsers name:string --connection test');
+
+        $this->assertExitCode(BaseCommand::CODE_ERROR);
+        $this->assertErrorContains('Migration name should start with one of the following prefixes: Create, Drop, Add, Remove, Alter.');
```
Comment (dereuromark) [inline tests/TestCase/Command/BakeMigrationCommandTest.php]:
```suggestion
        $this->assertErrorContains('When applying fields the migration name should start with one of the following prefixes: `Create`, `Drop`, `Add`, `Remove`, `Alter`.');
```

File: src/Command/BakeMigrationCommand.php (hunk 2 above)
Comment (rrd108) [reply src/Command/BakeMigrationCommand.php:93]: well. That is the original code as it is now in the 4.x branch.
Should I change it?

File: src/Command/BakeMigrationCommand.php (hunk 1 above)
Comment (rrd108) [reply src/Command/BakeMigrationCommand.php]: ok

File: src/Command/BakeMigrationCommand.php (hunk 2 above)
Comment (rrd108) [reply src/Command/BakeMigrationCommand.php:93]: Ahyou ment the message not replacing `empty()`. Done

**Pull request merged**